app/enterprise/0.35-x/kong-manager/security.md

```diff
@@ -1,7 +1,14 @@
 ---
 title: Securing Kong Manager
 book: admin_gui
+toc: false
 ---
+#### Table of Contents
```

Comment:
What issues are you having with the generated ToC?

Reply:
It was not appearing locally, then got duplicated on the staging site. Unable to replicate it now, so I'll remove the manual one and remove `toc: false`